src/patternfly/components/Table/table-tree-view.scss

```diff
@@ -59,7 +60,14 @@ $pf-c-tree-view--MaxDepth: 10;
   }
 
   > .pf-c-table__check {
-    margin-right: var(--pf-c-table__tree-view-main--c-table__check--MarginRight);
+    align-self: stretch;
+    margin: calc(var(--pf-c-table--cell--PaddingTop) * -1) 0 calc(var(--pf-c-table--cell--PaddingBottom) * -1) 0;
+  }
+
+  > .pf-c-table__check label {
+    height: 100%;
```

Comment:
This could be a little weird that the target area is so tall, and doesn't match the expandable toggle:

<img width="392" alt="Screen Shot 2022-01-26 at 2 28 51 PM" src="https://user-images.githubusercontent.com/35148959/151242001-6707e10d-3c3e-4a76-a6d2-d2ea17f267ec.png">

expandable toggle

<img width="479" alt="Screen Shot 2022-01-26 at 2 28 55 PM" src="https://user-images.githubusercontent.com/35148959/151241998-de26529f-736d-4f28-894b-0c82f7f33527.png">